tests/cecil-tests/Helper.cs

```diff
@@ -219,6 +224,139 @@ public static IEnumerable<ICustomAttributeProvider> EnumerateAttributeProviders
 				yield return item;
 		}
 
+		public static IEnumerable<MemberReference> EnumerateMembers (this AssemblyDefinition assembly, Func<MemberReference, bool>? filter = null)
+		{
+			foreach (var item in assembly.EnumerateTypes (filter))
+				yield return item;
+
+			foreach (var item in assembly.EnumerateFields (filter))
+				yield return item;
+
+			foreach (var item in assembly.EnumerateMethods (filter))
+				yield return item;
+
+			foreach (var item in assembly.EnumerateProperties (filter))
+				yield return item;
+
+			foreach (var item in assembly.EnumerateEvents (filter))
+				yield return item;
```

Comment:
You can use Concat from Linq here.

Reply:
That doesn't really line up all that nicely...